fedimint-client/src/oplog.rs

```diff
@@ -64,45 +67,107 @@ impl OperationLog {
         limit: usize,
         start_after: Option<ChronologicalOperationLogKey>,
```

Comment:
"verbs" as variable names, confusing. Also, "after" suggests `<`, while for range start I would expect `start <= x`

Reply:
`last_seen` maybe?

Reply:
Would rename this function too while at it.

Reply:
> Would rename this function too while at it.

Please make a recommendation in that case :laughing:

Reply:
`get_operations_before_op_log_key`? `paginate_operations_rev`? `paginate_oplog_desc`?

Reply:
As discussed on our call I split up the function to make it more readable.